runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java

```diff
@@ -119,7 +119,65 @@
+	ZENYTE(Type.GEM, "Zenyte", "Z", ItemID.UNCUT_ZENYTE, ItemID.ZENYTE),
+
+	//Nets
+	SMALL(Type.NET, "Small", "S", ItemID.SMALL_FISHING_NET),
+	BIG_NET(Type.NET, "Big", "B", ItemID.BIG_FISHING_NET),
+	DRIFT(Type.NET, "Drift", "D", ItemID.DRIFT_NET),
+
+	//Greegrees
+	ANCIENT(Type.GREEGREE, "Ancient", "A", ItemID.ANCIENT_GORILLA_GREEGREE),
+	BEARDED(Type.GREEGREE, "Bearded", "B", ItemID.BEARDED_GORILLA_GREEGREE),
+	GORILLA(Type.GREEGREE, "Gorilla", "G", ItemID.GORILLA_GREEGREE),
+	KARAMJAN(Type.GREEGREE, "Karamja", "K", ItemID.KARAMJAN_MONKEY_GREEGREE),
+	KRUK(Type.GREEGREE, "Kruk", "KR", ItemID.KRUK_MONKEY_GREEGREE),
+	SMALL_NINJA(Type.GREEGREE, "S Ninja", "SN", ItemID.NINJA_MONKEY_GREEGREE),
+	MEDIUM_NINJA(Type.GREEGREE, "M Ninja", "MN", ItemID.NINJA_MONKEY_GREEGREE_4025),
+	SMALL_ZOMBIE(Type.GREEGREE, "S Zombie", "SZ", ItemID.ZOMBIE_MONKEY_GREEGREE),
+	BIG_ZOMBIE(Type.GREEGREE, "L Zombie", "LZ", ItemID.ZOMBIE_MONKEY_GREEGREE_4030),
+
+	//Bones
+	BABYDRAGON(Type.BONE, "Baby drag", "BD", ItemID.BABYDRAGON_BONES),
+	BAT(Type.BONE, "Bat", "BAT", ItemID.BAT_BONES),
+	BIG_BONE(Type.BONE, "Big", "BIG", ItemID.BIG_BONES),
+	BONES(Type.BONE, "Normal", "N", ItemID.BONES),
+	BURNT(Type.BONE, "Burnt", "BU", ItemID.BURNT_BONES),
+	CHEWED(Type.BONE, "Chewed", "CH", ItemID.CHEWED_BONES),
+	DAGANNOTH(Type.BONE, "Dagan", "DA", ItemID.DAGANNOTH_BONES),
+	DRAGON(Type.BONE, "Dragon", "D", ItemID.DRAGON_BONES),
+	DRAKE(Type.BONE, "Drake", "DR", ItemID.DRAKE_BONES),
+	FAYRG(Type.BONE, "Fayrg", "F", ItemID.FAYRG_BONES),
+	HYDRA(Type.BONE, "Hydra", "H", ItemID.HYDRA_BONES),
+	JOGRE(Type.BONE, "Jogre", "J", ItemID.JOGRE_BONES),
+	LAVA_DRAGON(Type.BONE, "Lava", "L", ItemID.LAVA_DRAGON_BONES),
+	MANGLED(Type.BONE, "Mangled", "M", ItemID.MANGLED_BONES),
+	KARAMJA(Type.BONE, "Karamja", "K", ItemID.MONKEY_BONES),
+	SMALL_ZOMBIE_BONE(Type.BONE, "S Zombie", "SZ", ItemID.MONKEY_BONES_3180),
+	LARGE_ZOMBIE(Type.BONE, "L Zombie", "LZ", ItemID.MONKEY_BONES_3181),
+	GORILLA_BONE(Type.BONE, "Gorilla", "G", ItemID.MONKEY_BONES_3182),
+	BEARDED_BONE(Type.BONE, "Bearded", "B", ItemID.MONKEY_BONES_3183),
+	SMALL_NINJA_BONE(Type.BONE, "S Ninja", "SN", ItemID.MONKEY_BONES_3185),
+	MEDIUM_NINJA_BONE(Type.BONE, "M Ninja", "MN", ItemID.MONKEY_BONES_3186),
+	OURG(Type.BONE, "Ourg", "O", ItemID.OURG_BONES),
+	RAURG(Type.BONE, "Raurg", "R", ItemID.RAURG_BONES),
+	SHAIKAHAN(Type.BONE, "Shaika", "SH", ItemID.SHAIKAHAN_BONES),
+	SHEEP_1(Type.BONE, "Sheep 1", "S1", ItemID.SHEEP_BONES_1),
+	SHEEP_2(Type.BONE, "Sheep 2", "S2", ItemID.SHEEP_BONES_2),
+	SHEEP_3(Type.BONE, "Sheep 3", "S3", ItemID.SHEEP_BONES_3),
+	SHEEP_4(Type.BONE, "Sheep 4", "S4", ItemID.SHEEP_BONES_4),
+	SUPERIOR_DRAGON(Type.BONE, "Sup Drag", "SD", ItemID.SUPERIOR_DRAGON_BONES),
+	WOLF(Type.BONE, "Wolf", "WF", ItemID.WOLF_BONES),
+	WYRM(Type.BONE, "Wyrm", "WR", ItemID.WYRM_BONES),
+	WYVERN(Type.BONE, "Wyvern", "WV", ItemID.WYVERN_BONES),
+	ZOGRE(Type.BONE, "Zogre", "Z", ItemID.ZOGRE_BONES),
+
+	//Waterskins
```

Comment:
I think waterskins would make more sense in the item charges plugin

Reply:
Yes I thought so too but appearently the user wanted it for seeing which waterskins hold which charges in their bank.
If this is superflous it could be removed.